pkg/monitortests/kubeapiserver/auditloganalyzer/monitortest.go

```diff
@@ -122,6 +125,35 @@ func (w *auditLogAnalyzer) EvaluateTestsFromConstructedIntervals(ctx context.Con
 
 	}
 
+	for userAgent, panics := range w.apiserverPaniced.panicEventsPerUserAgent {
+		testName := fmt.Sprintf("user %s must not produce too many apiserver handler panics", userAgent)
+
+		failures := []string{}
+		if len(panics) > 5 {
+			ids := []string{}
+			for _, id := range panics.UnsortedList() {
+				ids = append(ids, string(id))
+			}
+			failures := append(failures, fmt.Sprintf("failed auditIDs: %s", strings.Join(ids, ",")))
```

Comment:
we need times for these to track them more effectively.

Reply:
we store audit-id, so later more info - like starting time - can be extracted. Do we want time to do more complicated alerting like "no more that n panics in x minutes"?

Reply:
> Do we want time to do more complicated alerting like "no more that n panics in x minutes"?

I don't think these are panics.  They may look like panics, but that's because the timeout handler prints stacks.

>we store audit-id, so later more info - like starting time - can be extracted.
you store the entire auditEvent to pull the times out.  It's just a lot more useful to see it on the spyglass page and compare it to "what's  happening then" than to download an audit log and parse backwards.